src/plots/hoverformat_attributes.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var FORMAT_LINK = require('../constants/docs').FORMAT_LINK;
+
+module.exports = function axisHoverFormat(x, mockedAxis) {
+    return {
+        valType: 'string',
+        dflt: '',
+        editType: 'none',
+        description: [
+            'Sets the hover text formatting rule for `' + x + '`',
+            ' using d3 formatting mini-languages which are very similar to those in Python.',
+            'See: ' + FORMAT_LINK,
```

Comment:
Nice job exporting this just as a function. But it still needs the option to include dates when they're supported.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/5563/commits/8c33d50a8daabab795fc29bcd545342ebaae13c2.